modules/sdk/src/main/scala/org/tessellation/sdk/app/TessellationIOApp.scala

```diff
@@ -159,6 +167,12 @@ abstract class TessellationIOApp[A <: CliMethod](
                             )
                             .asResource
 
+                          maybeMajorityPeerIds <- getMajorityPeerIds(
```

Comment:
afaik Ryle said that GL0 should be untouched, so we won't be using this on Global Layer 0, so IMO it should be a separate method executed from DAG L1, CL0 and CL1 (whatever required) but not by default in TessellationIOApp (so on GL0 as well)

Reply:
@marcinwadon Are you ok with `core.Main` providing an overridden `getMajorityPeerIds` that always returns `None` ?

Reply:
instead of overriding it, you may not provide it at all wherever it's not needed 🤔

Reply:
@marcinwadon Yes I thought of that too. What I don't know is what is preferred by the team: overriding it in the one place we don't want it, or duplicating the code 3 times in the place we do.